go/worker/compute/executor/committee/node.go

```diff
@@ -511,7 +518,22 @@ func (n *Node) startProcessingBatchLocked(
 			batchRuntimeProcessingTime.With(n.getMetricLabels()).Observe(time.Since(rtStartTime).Seconds())
 		}()
 
-		rsp, err := rt.Call(ctx, rq)
+		roundCtx := n.commonNode.Group.RoundContext()
+		var rsp *protocol.Body
+		call := func() error {
+			var grr error
+			rsp, grr = rt.Call(ctx, rq)
```

Comment:
Why did you add a retry here? This should not be needed.

The only methods that need retry are (as mentioned in #3000 with links):
* `HandleBatchFromTransactionSchedulerLocked` in executor.
* `HandleResultsFromExecutorWorkerLocked` in merge.

Reply:
`HandleBatchFromTransactionSchedulerLocked` calls `maybeStartProcessingBatchLocked`, which calls `startProcessingBatchLocked`, which can fail only in this `rt.Call`.  There doesn't seem to be any other place to put the retry but here.

Reply:
Retrying here does not make sense as the `Call` internally includes a retry in case the runtime is not available. If it can't fail elsewhere, maybe retry is not needed for the transaction scheduler case then? I'll take another look.

Reply:
Yeah it seems that `HandleBatchFromTransactionSchedulerLocked` does not need retry.

Reply:
OK, I'll remove it then :)